sample-apps/discount-functions-sample-app/web/frontend/vite.config.js

```diff
@@ -1,15 +1,14 @@
-import { defineConfig } from "vite";
-import { dirname } from "path";
-import { fileURLToPath } from "url";
+import { defineConfig, loadEnv } from 'vite';
```

Comment:
Just curious: were these formatting changes made by Prettier?

Reply:
Yes. If you look at `frontend/package.json` you will see the that the prettier config is set there and it's set to `"singleQuote": true`